templates/leaderboard/leaderboard.html

```diff
@@ -9,24 +9,76 @@
 {% endblock %}
 
 {% block content %}
-    <h2>Investments</h2>
-    <div class="break">
-        <hr>
-    </div>
     <div class="center">
-        {% include "leaderboard/includes/list.html" with name='Artists' leaders=top_raised %}
-        {% include "leaderboard/includes/list.html" with name='Investors' leaders=top_invested %}
-    </div>
-    <h2>Earnings</h2>
-    <div class="break">
-        <hr>
-    </div>
-    <div class="center">
-        {% include "leaderboard/includes/list.html" with name='Artists' leaders=top_earned_artists %}
-        {% include "leaderboard/includes/list.html" with name='Investors' leaders=top_earned_investors %}
+        {% include "leaderboard/includes/list.html" %}
     </div>
 {% endblock %}
 
 {% block extrajs %}
     <script type="text/javascript" src="{% static "js/vendor/Chart.min.js" %}"></script>
+
+    <script>
+        var ctx = document.getElementById("InvestorChart").getContext('2d');
+        var myChart = new Chart(ctx, {
+            type: 'bar',
+            data: {
+                labels: [
+                    {% if top_earned_investors %}
+                        {% for top_earned_investors_context in top_earned_investors %}
+                            "{{ top_earned_investors_context.name }}"{% if not forloop.last %},{% endif %}
+                        {% endfor %}
+                    {% endif %}],
```

Comment:
The indenting is a lot better, but we could make it better in a few places. Here you have a close bracket at the end when I think a hanging bracket would be better:
```javascript
labels: [
    {% if top_earned_investors %}
        ...
    {% endif %}
],
datasets: [
    {
        label: 'Amount Invested',
        ...
    },
    {
        ...
```
Also note the indentation of `datasets`.